flagbit/category-bundle
# Implement data transformers for custom property types

Implement data transformers that allow to modify data during normalization and denormalization. This enables custom properties with custom data formats that could not be handled by the default import/export.

Test Cases

- Check if the category import sill works and imports data as expected.
- Check if the category export still works and exports data  as expected
- Check the import and export with a "checkbox" that uses a custom transformer by default
  - Import works as expected
  - Export works as expected
